src/DynamoMLDataPipeline/DynamoMLDataPipeline.cs

```diff
@@ -49,34 +49,53 @@ internal string ProductionCollectionID
             }
         }
 
-        internal static IOAuth2AccessTokenProvider AuthTokenProvider { get; set; }
+        internal DynamoModel DynamoModel { get; set; }
+
+        internal static IDSDKManager AuthProvider { get; set; }
+
+        // Id of the user sending the post request.
+        private static string UserId
+        {
+            get
+            {
+                return AuthProvider.UserId;
```

Comment:
Have you tried this in Revit? You have not added a new interface, and implemented it to make sure that ALL of of the auth providers actually have this method.

In Revit, `IDSDKManager` type will not be loaded, that's only in sandbox.

Reply:
Yes, this was one of the questions I had before testing on Revit. I will look into this.

Reply:
Works fine in Revit.